### PR TITLE
Add info for initialMessages

### PR DESCRIPTION
**Pull request by benlower**

### CODE REVIEW

File: docs/src/content/docs/api/calls.mdx [hunk 1]
```diff
@@ -531,4 +536,50 @@ Call recordings are only generated if you add `"recordingEnabled": true` to the
...
+    "role": "MESSAGE_ROLE_AGENT",
+    "text": "Great to meet you, Steve! How can I help?"
+  },
+]
+```
+
+#### Using Mistral
+If you are using `fixie-ai/ultravox-mistral-nemo-12B` as your model, you need to do the following when creating the call:
+1. **Empty System Prompt** → Set `systemPrompt` to an empty string.
+1. **Prompt in Initial Messages** → Add a single user message to `initialMessages` that contains the system prompt.
```
Comment (mdepinet) [inline docs/src/content/docs/api/calls.mdx:569]: @petersalas Is this right?  Seems like we should just do this in the chat template.

File: docs/src/content/docs/api/calls.mdx [hunk 2]
```diff
@@ -531,4 +536,50 @@ Call recordings are only generated if you add `"recordingEnabled": true` to the
...
+    "text": "My name is Steve"
+  },
+  {
+    "role": "MESSAGE_ROLE_AGENT",
+    "text": "Great to meet you, Steve! How can I help?"
+  },
+]
+```
+
+#### Using Mistral
```
Comment (mdepinet) [inline docs/src/content/docs/api/calls.mdx:566]: Should also mention that Mistral requires a strict alternating user-agent message order.

File: docs/src/content/docs/api/calls.mdx [hunk 3]
```diff
@@ -531,4 +536,50 @@ Call recordings are only generated if you add `"recordingEnabled": true` to the
...
-</Tabs>
\ No newline at end of file
+</Tabs>
+
+## More Info
+
+This section contains additional details for some properties.
+
+### initialMessages
+When creating a new call or a new call stage, you can provide messages to the agent via `initialMessages`. By default, new calls don't have initial messages and call stages inherit the prior stage's messages.
```
[the remark this answers is not in the document]
Comment (benlower) [reply docs/src/content/docs/api/calls.mdx:546]: good call. added.

File: docs/src/content/docs/api/calls.mdx [hunk 4]
```diff
@@ -531,4 +536,50 @@ Call recordings are only generated if you add `"recordingEnabled": true` to the
...
+
+These messages can serve the purpose of giving the agent call history or to give examples for few-shotting (e.g. if you want the agent to learn how to respond in a specific way to user input).
+
+#### Message Format
+`initialMessages` must be an array of message objects where each message contains a `role` and `text`. `role` must be either `MESSAGE_ROLE_USER` or `MESSAGE_ROLE_AGENT`. Here's an example:
```
[the remark this answers is not in the document]
Comment (benlower) [reply docs/src/content/docs/api/calls.mdx:551]: @mdepinet you mean `ultravox.v1.Message` in swagger file?

File: docs/src/content/docs/api/calls.mdx (hunk 2 above)
Comment (benlower) [reply docs/src/content/docs/api/calls.mdx:566]: added.

File: docs/src/content/docs/api/calls.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (mdepinet) [reply docs/src/content/docs/api/calls.mdx:551]: Yes, but also that this is already documented: https://docs.ultravox.ai/api/calls/

[the remark this answers is not in the document]
Comment (benlower) [reply docs/src/content/docs/api/calls.mdx:551]: ah yes

File: docs/src/content/docs/api/calls.mdx (hunk 1 above)
Comment (benlower) [reply docs/src/content/docs/api/calls.mdx:569]: i'm going to merge this. @petersalas LMK if we need to make any updates.

**Pull request merged**